teamscale-gradle-plugin/src/main/kotlin/com/teamscale/config/JacocoReportConfiguration.kt

```diff
@@ -2,12 +2,23 @@ package com.teamscale.config
 
 import com.teamscale.client.EReportFormat
 import org.gradle.api.Project
+import org.gradle.api.file.FileCollection
+import org.gradle.api.file.RegularFileProperty
 import org.gradle.testing.jacoco.tasks.JacocoReport
+import java.io.File
 
 /** Configuration for the JaCoCo coverage report. */
 open class JacocoReportConfiguration(project: Project, task: JacocoReport) :
     ReportConfigurationBase(EReportFormat.JACOCO, project, task) {
-    init {
-        destination.set(task.reports.xml.outputLocation)
+    /** The destination where the report should be written to/read from. */
+    var destination: RegularFileProperty = task.reports.xml.outputLocation
+
+    /** Allows to set the destination as a string, which is resolved as File internally. */
+    fun setDestination(destination: String) {
+        this.destination.set(project.objects.fileProperty().fileValue(File(destination)))
+    }
```

Comment:
Not sure if this is done before, but if not, shouldn't we somehow handle wrong destinations here? Like destination is a directory, not readable/writable?